src/common/reduction/matching/test/GapMatcherTest.cpp

```diff
@@ -1,13 +1,54 @@
-/** Copyright (C) 2017 European Spallation Source ERIC */
+/** Copyright (C) 2017-2022 European Spallation Source ERIC */
```

Comment:
c++ comments but also add spaces between dash and the latest version that includes reference to LICENSE